pyartcd/pyartcd/pipelines/prepare_release.py

```diff
@@ -270,6 +272,14 @@ async def run(self):
             except CalledProcessError as ex:
                 _LOGGER.warning(f"Unable to move {impetus} advisory {advisory} to QE: {ex}")
 
+        _LOGGER.info("Check failed greenwave tests...")
+        self._slack_client.bind_channel(self.release_name)
+        for advisory in [advisories["image"], advisories["extras"], advisories["metadata"]]:
+            test_result = ErrataConnector()._get(f'/api/v1/external_tests?filter[test_type]=greenwave_cvp&filter[status]=FAILED&filter[active]=true&page[size]=1000&filter[errata_id]={advisory}')
```

Comment:
from what I see the greenwave result only valuable here, so didn't relay on elliott